tensorflow/lite/tools/verifier.cc

```diff
@@ -50,8 +50,12 @@ void ReportError(ErrorReporter* error_reporter, const char* format, ...) {
   }
 }
 // Returns the int32_t value pointed by ptr.
-const uint32_t* GetIntPtr(const char* ptr) {
-  return reinterpret_cast<const uint32_t*>(ptr);
+const uint32_t GetIntPtr(const char* ptr) {
+#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
```

Comment:
Not an expert on comiler flags, but it seems that this is not supported on all compilers. Could you add more flags taking abseil as a reference?

https://github.com/abseil/abseil-cpp/blob/5d8fc9192245f0ea67094af57399d7931d6bd53f/absl/base/config.h#L452

Reply:
Yeah I can add some additional flag checks here. Even inside the TensorFlow codebase there have been a variaty of macro implementations to check endianness, but I think checking `__BYTE_ORDER__` is defined then checking the value is `__ORDER_BIG_ENDIAN__` should cover most compilers.